lib/psql.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const co = require('co')
+const debug = require('debug')('psql')
+
+function * exec (db, query) {
+  const execa = require('execa')
+  const os = require('os')
+  const cli = require('heroku-cli-util')
+
+  debug(query)
+  let env = {
+    PGAPPNAME: 'psql non-interactive',
```

Comment:
yeah this is only partial functionality, and it isn't actually adding anything to this plugin. It's just used by heroku-pg-extras that imports this. I decided to put it here since I know I'll reuse the logic soon for the heroku-pg commands that need to run remote queries.

I still need to write the interactive version for `pg:psql`.